docs/hub/storage-regions.md

```diff
@@ -1,53 +1,53 @@
 # Storage Regions on the Hub
 
-Regions let you decide where your org's models and datasets will be stored.
+Regions allow you to specify where your organization's models and datasets are stored.
 
 <Tip warning={true}>
 This feature is part of the <a href="https://huggingface.co/enterprise" target="_blank">Enterprise Hub</a>.
 </Tip>
 
-This has two main benefits:
+This offers two key benefits:
 
 - Regulatory and legal compliance
-- Performance (improved download and upload speeds and latency)
+- Performance (faster download/upload speeds and lower latency)
 
-Currently we support the following regions:
+Currently available regions:
 
 - US 🇺🇸
 - EU 🇪🇺
-- coming soon: Asia-Pacific 🌏
+- Coming soon: Asia-Pacific 🌏
 
 ## How to set up
 
-If your organization is subscribed to Enterprise Hub, you will be able to see the Regions settings page:
+Organizations subscribed to Enterprise Hub can access the Regions settings page:
 
 ![](https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/hub/storage-regions/feature-annotated.png)
 
-On that page you can see:
+This page displays:
 
-- an audit of where your organization repos are currently located
-- dropdowns to select where your repos will be created
+- An audit of your organization's repository locations
+- Options to select where new repositories will be stored
 
 ## Repository Tag
 
-Any repo (model or dataset) stored in a non-default location will display its Region directly as a tag. That way your organization's members can see at a glance where repos are located.
+Any repository (model or dataset) stored in a non-default location displays its Region as a tag, allowing organization members to quickly identify repository locations.
 
 <div class="flex justify-center">
 <img class="block" width="400" src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/hub/storage-regions/tag-on-repo.png"/>
 </div>
 
 ## Regulatory and legal compliance
 
-In regulated industries, companies may be required to store data in a specific region.
+Regulated industries often require data storage in specific regions.
 
-For companies in the EU, that means you can use the Hub to build ML in a GDPR compliant way: with datasets, models and inference endpoints all stored within EU data centers.
+For EU companies, this means you can use the Hub for ML development in a GDPR-compliant manner, with datasets, models and inference endpoints stored in EU data centers.
 
 ## Performance
 
-Storing your models or your datasets closer to your team and infrastructure also means significantly improved performance, for both uploads and downloads.
+Storing models and datasets closer to your team and infrastructure significantly improves performance for both uploads and downloads.
 
-This makes a big difference considering model weights and dataset files are usually very large.
+This impact is substantial given the typically large size of model weights and dataset files.
```

Comment:
```suggestion
This impact is substantial, given the large model weights and dataset files.
```

Reply:
it's not always large so it's fine I think.